src/glue/resources/table_columns.yaml

```diff
@@ -1,4 +1,49 @@
 tables:
+  3DRotation_v1:
```

Comment:
Remind me again - how do we find the data types of the fields here for BD? Just through the Glue Crawler or parsing through the production 3D rotation data in Synapse ourselves (which would have all of the expected fields in one file or do you have to look through multiple)?

Reply:
Initially, we derive the data types from the JSON Schema provided for the assessment. These Glue table schemas can end up evolving beyond any single JSON Schema (for example, if the assessment developers put out a new revision of the 3DRotation assessment with an additional field and remove other fields), but that hasn't happened IRL yet.